lightwood/data/splitter.py

```diff
@@ -7,6 +7,9 @@
 
 
 def splitter(data: pd.DataFrame, k: int, tss: TimeseriesSettings) -> List[pd.DataFrame]:
+    """
+    Splits a dataframe into k equally-sized folds.
```

Comment:
I hate the term folds here as it's not clear what "folds" should be.

Are folds equally sized to the training data? are folds subsets of the data? Please clarify.

Reply:
Indeed, equally sized. As discussed elsewhere, will be renaming to `subset`.